test00.md

```diff
@@ -0,0 +1,46 @@
+# Task 0: Get familiar with Git and GitHub [![Gitter](https://badges.gitter.im/Kottans/frontend.svg)](https://gitter.im/Kottans/frontend?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+1. Finish the course [How to use Git and GitHub](https://www.udacity.com/course/how-to-use-git-and-github--ud775)
+
+2. Finish [try.github.io](https://try.github.io/levels/1/challenges/1)
+
+3. Create repository named `kottans_frontend`.
+
+4. Create `README.md` for the repository.
+
+5. Describe your impressions about learned materials.
+
+6. Send a pull-request to repo [Kottans/mock-repo](https://github.com/Kottans/mock-repo) proposing a change. It could be anything, really. If you have found a typo in README - great! Take care to give your PR a meaningful name and description.
+**Please note, that your PR may not be reviewed quickly.**
+
+When you finish this task you can start [the next one](test01.md).
+
+### P.S. You should know
```

Comment:
How about putting this section back to README.md? It refers to the entire course.